securedrop/tests/functional/functional_test.py

```diff
@@ -226,12 +304,6 @@ def setup(self, session_expiration=30):
         #
         self.driver.implicitly_wait(15)
 
-
-        # Set window size and position explicitly to avoid potential bugs due
-        # to discrepancies between environments.
-        #self.driver.set_window_position(0, 0)
```

Comment:
Is it necessary to remove this? Standardizing this prevents us from having divergent window sizes between CI and dev that can produce CI test failures that are annoying to debug

Reply:
We can get these back into the code.

Reply:
I finally figured out why i had to comment that out. https://github.com/mozilla/geckodriver/issues/676 is causing the trouble as we have to test against the `firefox 52esr` in this branch. None of those position calls are working in this. @redshiftzero